sktime/decomposition/vamp.py

```diff
@@ -347,6 +338,42 @@ def _decomposition(self) -> _DiagonalizationResults:
             rank0=rank0, rankt=rankt, singular_values=singular_values, left_singular_vecs=U, right_singular_vecs=V
         )
 
+    def forward(self, trajectory: np.ndarray, component: Optional[int] = None):
+        r"""Applies the forward transform to the trajectory in non-whitened space. This is achieved by
+        transforming each frame :math:`X_t` with
+
+        .. math::
+            \hat{X}_{t+\tau} = (V^\top)^{-1} \Sigma U^\top (X_t - \mu_0) + \mu_t,
+
+        where :math:`V` are the left singular vectors, :math:`\Sigma` the singular values, and :math:`U` the right
+        singular vectors.
+
+        Parameters
+        ----------
+        trajectory : (T, n) ndarray
+            The input trajectory
+        component : int or None
+            The component to project onto. If None, all processes are taken into account, if integer sets all singular
+            values to zero but the "component"th one.
```

Comment:
is there ever an interest in using more than one but less than all

Reply:
perhaps for superposition of components, ill add it to the implementation

Reply:
seems straightforward to add, right?

Reply:
quite straightforward, added and also modified the vamp notebook a bit to showcase a usage of multiple components

Reply:
this is great!